Rohit/PostgresAssignment/myschool.sql

```diff
@@ -0,0 +1,123 @@
+create schema myschool;
+set search_path to myschool;
+
+create table students(studentId int not null primary key,
+					  name text not null ,
+					  dob date not null,
+					 gender char null);
+					 
+insert into students values('1','anurag','1997-02-04','M');
+insert into students values('2','niti','1999-02-06','F');
+insert into students values('3','ayog','1998-02-07','M');
+insert into students values('4','anup','1995-02-06','M');
+insert into students values('5','chirag','1997-05-04','M');
+insert into students values('6','rahul','1987-05-08','M');
+insert into students values('7','kaaju','1996-02-08','F');
+insert into students values('8','jaggu','1997-07-23','F');
+insert into students values('9','prince','1997-05-13','M');
+insert into students values('10','yash','2000-02-04','M');
+
+create table teachers(teacherId int not null primary key,
+					teacher_fname text not null,
```

Comment:
you can get in touch with jatin to fix all your schemas - data types.

Reply:
resolved as in place of text ,used varchar